samples/application_development/sysbuild/with_mcuboot/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+    description: Sample with MCUboot built through sysbuild
+    name: with mcuboot
+tests:
+  sample.application_development.sysbuild.with_mcuboot:
+    sysbuild: True
+    # Platform allowed is used as twister using sysbuild still lacks proper
+    # filtering support, see discussion in #49552.
+    platform_allow: reel_board nrf52840dk_nrf52840
+    tags: mcuboot
```

Comment:
Would need to force the `--sysbuild` argument here for the test system, otherwise it will be build without it/mcuboot

Reply:
No, `--sysbuild` is guarenteed by this line:
https://github.com/zephyrproject-rtos/zephyr/blob/482baa41b9bb538e13466f56211d885892d9f72d/samples/application_development/sysbuild/with_mcuboot/sample.yaml#L6